### PR TITLE
IGNITE-16445 Always specify charset explicitly

### PR DESCRIPTION
**Pull request by rpuch**
https://issues.apache.org/jira/browse/IGNITE-16445

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -109,6 +109,7 @@
         <maven.source.plugin.version>3.2.1</maven.source.plugin.version>
         <maven.surefire.plugin.version>3.0.0-M5</maven.surefire.plugin.version>
         <maven.exec.plugin.version>3.0.0</maven.exec.plugin.version>
+        <maven.modernizer.plugin.version>2.3.0</maven.modernizer.plugin.version>
```
Comment (vveider) [inline parent/pom.xml:112]: Plugin versions should be ABC sorted

Comment (vveider) [inline parent/pom.xml:112]: Plugin version should be ABC sorted

**Review by vveider — CHANGES_REQUESTED** [review]

File: parent/pom.xml (hunk 1 above)
Comment (rpuch) [reply parent/pom.xml]: Fixed

**Review by AMashenkov — APPROVED** [review]

File: parent/pom.xml [hunk 2]
```diff
@@ -854,6 +854,49 @@
                 </plugins>
             </build>
         </profile>
+
+        <!--
+            This profile is used to check the code for legacy API.
+
+            It runs by default and can be skipped with either
+            -Dmaven.chechstyle.skip or -Dmodernize.skip flag.
```
[the remark this answers is not in the document]
Comment (vveider) [reply parent/pom.xml]: Fixed that.

**Review by vveider — APPROVED** [review]

**Pull request merged**